src/modules/Elsa.MassTransit/Features/MassTransitFeature.cs

```diff
@@ -53,21 +55,42 @@ public override void Apply()
 
         void Configurator(IBusRegistrationConfigurator configure)
         {
-            configure.UsingInMemory((context, configurator) =>
-            {
-                configurator.SetupWorkflowDispatcherEndpoints(context);
-                configurator.ConfigureEndpoints(context);
+            var consumers = this.GetConsumers().ToList();
+            var temporaryConsumers = consumers
+                .Where(c => c.IsTemporary)
+                .ToList();
 
-                if (PrefetchCount != null) configurator.PrefetchCount = PrefetchCount.Value;
+            configure.UsingInMemory((context, busFactoryConfigurator) =>
+            {
+                var options = context.GetRequiredService<IOptions<MassTransitWorkflowDispatcherOptions>>().Value;
+
+                foreach (var consumer in temporaryConsumers)
```

Comment:
How will this work when we have the ASB enabled? Does this config get overridden or will it create both InMemory and ASB Endpoints?

Reply:
Notice that this logic sits inside of a local function, that gets assigned to the BusConfigurator lambda property conditionally:

<img width="1531" alt="image" src="https://github.com/elsa-workflows/elsa-core/assets/938393/cb815aee-79df-40ce-b37e-8c26ac6558be">

That being said, I'll see if I can just move it outside of this method and make it a separate, private method for improved readability.